app/views/late_reservation_mailer/delinquent_reservation.html.slim

```diff
@@ -0,0 +1,7 @@
+h2 Your gifts were due today (#{Date.today.strftime("%B %d, %Y")}).
+p
+  | At least one of the gifts you reserved has not yet been received by Wediko.
+    If you shipped them recently, there's no reason to worry, but if you have yet to ship your gifts, please log in below to mark them as shipped.
```

Comment:
Now that you mention it, you can't actually mess with reservations once they're marked as delinquent. Let me fix that (and add a line to the email).